CONTRIBUTING.md

```diff
@@ -0,0 +1,504 @@
+# AWS iOS SDK Contributing Guide
+
+Thank you for your interest in contributing to our project! <3 Whether it's a bug report, new feature, correction, or additional documentation, we greatly value feedback and contributions from our community. Please read through it carefully before submitting a PR or issue and let us know if it's not up-to-date (or even better, submit a PR with your corrections ;)).
+
+- [Getting Started](#getting-started)
+- [Tools](#tools)
+  - [Integration Testing Set Up](#integration-testing-setup)
+- [Workflows](#workflows)
+  - [New Features or Bug Fixes](#New-Features-or-Bug-Fixes)
+- [Debugging](#debugging)
+  - [Running Cocoapods Locally](#running-cocoapods-locally)
+- [Pull Requests](#pull-requests)
+  - [Pull Request Checklist](#pull-request-checklist)
+  - [Step 1: Open Issue](#step-1-open-issue)
+  - [Step 2: Design (optional)](#step-2-design-optional)
+  - [Step 3: Fork The Repo](#step-3-fork-the-repo)
+  - [Step 4: Work your Magic](#step-3-work-your-magic)
+  - [Step 5: Commit](#step-4-commit)
+  - [Step 6: Pull Request](#step-5-pull-request)
+  - [Step 7: Merge](#step-6-merge)
+- [Troubleshooting](#troubleshooting)
+- [Related Repositories](#related-repositories)
+- [Finding Contributions](#finding-contributions-to-work-on)
+- [Code of Conduct](#code-of-conduct)
+- [Security Issue Notifications](#security-issue-notifications)
+- [Licensing](#licensing)
+
+## Getting Started
+
+To get started with contributing to the AWS iOS SDK, first make sure you have the latest version of [Xcode](https://developer.apple.com/xcode/) installed as well as [cocoapods](https://guides.cocoapods.org/using/getting-started.html). You can install cocoapods by simply running:
+
+```bash
+sudo gem install cocoapods
```

Comment:
Cocoapods is not required for our sdk development.

Reply:
I realize that but that's how we dev on it and I don't know how to instruct them to do it without it because I am not a carthage user. What would you rather this say?

Reply:
I can say there are other ways to get started but for the purpose of this instruction, im going to use cocoapods to help you get started?

Reply:
done